package.json

```diff
@@ -88,7 +88,7 @@
     "lodash-es": "^4.17.10",
     "nock": "^9.2.5",
     "preact": "^10.0.0",
-    "prettier": "1.18.2",
+    "prettier": "^1.19.1",
```

Comment:
Bumping this to the latest 1.x.x version because `1.18.2` fails on the nullish coalescing operator `??` operator